src/Combinatorics/Matroids/ChowRings.jl

```diff
@@ -210,11 +210,35 @@ function augmented_quadratic_relations(ring::MPolyRing, proper_flats::Vector{Vec
     return vcat(incomparable_polynomials, xy_polynomials)
 end
 
+@doc raw"""
+    volume_map(M::Matroid; A::MPolyQuoRing)
+
+Return A (normalized) function that maps the top degree component of the Chow ring to the base ring.
+
+# Examples
+The following computes the volume map of the Chow ring of the Fano matroid.
+```jldoctest
+julia> M = fano_matroid();
+
+julia> R = chow_ring(M);
+
+julia> f = volume_map(M,R);
+
+julia> f(R[1]*R[8])
+1
+
+julia> f(R[1]^2)
+-2
+```
 """
-A helper function to select indices of a vector that do `include` elements of a given set and `exclude` another
-"""
-function _select(include::Union{AbstractVector,Set},exclude::Union{AbstractVector,Set},set::Union{AbstractVector,Set})
-    all = union(set...)
-    compl = setdiff(all,exclude)
-    return findall(s->issubset(include,s)&&issubset(s,compl),set);
+function volume_map(M::Matroid, A::MPolyQuoRing)
+    flat = flats(M)[1]
+    prod = one(A)
+    for i in 2:length(flats(M))-1
+        if is_subset(flat, flats(M)[i])
+            flat = flats(M)[i]
+            prod*= A[i-1]
+        end
+    end
+    return f -> is_zero(f) ? 0//coeff(prod.f,1) : coeff(f.f,1)//coeff(prod.f,1)
```

Comment:
```suggestion
    return f -> is_zero(f) ? 0//coeff(lift(prod),1) : coeff(lift(f),1)//coeff(lift(prod),1)
```
Even though using the `.f` here might be well defined (which I cannot tell), I think it would be better to use some kind of accessor function instead of using the field directly.
While undocumented for this type, there is even a function which does exactly what you need:
https://github.com/oscar-system/Oscar.jl/blob/a31ce6c23cf5407265d371ec09cf71fa9823fc68/src/Rings/MPolyQuo.jl#L1014
(and `is_zero(f)` will do a `simplify` anyway)

Reply:
Thank you. I didn't knew the lift function exists